src/main/java/net/sf/jabref/logic/error/TeeStream.java

```diff
@@ -10,17 +12,24 @@
 public class TeeStream extends PrintStream {
 
     private final PrintStream outStream;
+    private final MessagePriority priority;
 
-    public TeeStream(PrintStream out1, PrintStream out2) {
+    public TeeStream(PrintStream out1, PrintStream out2, MessagePriority priority) {
         super(out1);
         this.outStream = out2;
+        this.priority = priority;
     }
 
     @Override
     public void write(byte[] buf, int off, int len) {
         try {
             super.write(buf, off, len);
             outStream.write(buf, off, len);
+            String s = new String(buf, off, len);
```

Comment:
I think `TeeStream` is the wrong place to add this code. There should be a central error logger somewhere. Sorry for being a bit unspecific about this, but I right now I have no time for digging into the details.